### PR TITLE
New: Add Scatter Graph

### PR DESCRIPTION
**Pull request by SharathAlur**
**What is the purpose of this pull request? (Check any that's applicable)**
Adding the ability to draw Scatter graph

-   [x] Feature
-   [ ] Bug fix
-   [ ] Dependency updates
-   [ ] Documentation updates
-   [ ] Build related changes
-   [ ] Changes an existing functionality
-   [ ] Other, please explain:

**Does this introduce a breaking change?**

-   [ ] Yes
-   [x] No

**What changes did you make? (Give an overview)**
Added a new graph type, scatter graph.

**Add any screenshots**
Screenshot 2019-12-17 at 8 46 22 PM

### CODE REVIEW

File: docs/controls/Scatter.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+            -   [Multi Scatter](#multi-scatter)
+    -   [JSON Properties](#json-properties)
+        -   [Root](#root)
+        -   [Data](#data)
+            -   [Required](#required)
+            -   [Optional](#optional)
+        -   [Values](#values)
+            -   [Required](#required-1)
+            -   [Optional](#optional-1)
+                -   [Style](#style)
```
Comment (abhijit945) [inline docs/controls/Scatter.md]: This is not valid.

File: docs/controls/Scatter.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
+# Scatter
```
Comment (abhijit945) [inline docs/controls/Scatter.md]: Can you link this graph in the same place as other graphs so that consumers can get to it better?

Comment (SharathAlur) [reply docs/controls/Scatter.md]: Added in [Readme.md](https://github.com/cerner/carbon-graphs/pull/128/commits/1d9e855ab0886f8a110f6e0eabb687c4dca4f18f#diff-04c6e90faac2675aa89e2176d2eec7d8R69), also removed it from [potential-feature](https://github.com/cerner/carbon-graphs/pull/128/commits/1d9e855ab0886f8a110f6e0eabb687c4dca4f18f#diff-04c6e90faac2675aa89e2176d2eec7d8L107) as the feature is added now!

File: docs/controls/Scatter.md (hunk 1 above)
Comment (SharathAlur) [reply docs/controls/Scatter.md]: Removed

File: dev/app.js [hunk 3]
```diff
@@ -115,6 +116,34 @@ import {
     renderTimelineNoXAxisTickLabel
 } from "./examples/controls/timeline";
 import { createElementLegendBindTo } from "./examples/helpers";
+import {
+    renderScatter,
+    renderScatterTimeSeries,
+    renderScatterWithDateline,
+    renderScatterY2Axis,
```
[the remark this answers is not in the document]
Comment (SharathAlur) [reply dev/app.js]: Removed some of the [examples](https://github.com/cerner/carbon-graphs/pull/128/commits/1d9e855ab0886f8a110f6e0eabb687c4dca4f18f#diff-1765d0005076c116a75098c1162e835bL123)

**Pull request merged**